data/src/main/java/co/nimblehq/compose/crypto/data/model/response/CoinPriceResponse.kt

```diff
@@ -0,0 +1,16 @@
+package co.nimblehq.compose.crypto.data.model.response
+
+import co.nimblehq.compose.crypto.domain.model.CoinPrice
+import com.squareup.moshi.Json
+
+data class CoinPriceResponse(
+    @Json(name = "prices")
+    val prices: List<List<Double>>
```

Comment:
Should we use BigDecimal to parse pricing data as we don't know how big of the price range? 🤔 https://github.com/nimblehq/jetpack-compose-crypto/pull/27/files#diff-38ebbcf13f3846ded8d564dd162a73024194a262034ba45d393ae125c3e7634d

Reply:
Makes sense a chai 👍

Reply:
@hoangnguyen92dn lol, you're so `ra zẻ`